arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -85,5 +85,28 @@ public async Task<GetCollectionCountResponse> GetCollectionCountAsync(GetCollect
                 throw await GetApiErrorException(response);
             };
         }
+
+        /// Get all collections
+        /// GET/_api/collection
+        /// </summary>
+        /// <param name="options"></param>
+        /// <returns></returns>
+        public async Task<GetCollectionsResponse> GetCollectionsAsync(GetCollectionsOptions options)
```

Comment:
I think it would be convenient to make `options` an optional argument, e.g. `GetCollectionsOptions options = null`